dev/bots/prepare_package.dart

```diff
@@ -17,12 +17,12 @@ import 'package:process/process.dart';
 
 const String chromiumRepo = 'https://chromium.googlesource.com/external/github.com/flutter/flutter';
 const String githubRepo = 'https://github.com/flutter/flutter.git';
-const String mingitForWindowsUrl = 'https://storage.googleapis.com/flutter_infra/mingit/'
+const String mingitForWindowsUrl = 'https://storage.googleapis.com/flutter_infra_release/mingit/'
     '603511c649b00bbef0a6122a827ac419b656bc19/mingit.zip';
-const String gsBase = 'gs://flutter_infra';
+const String gsBase = 'gs://flutter_infra_release';
```

Comment:
We also have to migrate the release manifests (e.g. `gs://flutter_infra/releases/releases_linux.json` etc) which are the source of truth for all published flutter releases which the website reads.

Reply:
Done.